client/src/views/AuthorIndividualForm.vue

```diff
@@ -138,14 +138,31 @@
             v-if="!$v.form.tmpAuthorYearOfBirth.minValue || !$v.form.tmpAuthorYearOfBirth.maxValue"
           >The year of birth must be between {{minAuthorYearOfBirth}} and {{maxAuthorYearOfBirth}}</span>
         </md-field>
+        <div ref="tmpAuthorYearOfDeath">
+          <md-field :class="getValidationClass('tmpAuthorYearOfDeath')">
+            <label for="author-year-of-death">Deceased</label>
+            <md-input
+              name="author-year-of-death"
+              id="author-year-of-death"
+              type="number"
+              v-model="form.tmpAuthorYearOfDeath"
+              :disabled="sending"
+              @input="updateField()"
+            />
+            <span
+              class="md-error"
+              v-if="!customValidationFields.tmpAuthorYearOfDeath.invalid"
```

Comment:
You did not use the customValidationFields, you used the validations "hook", so this should be something like v-if"!$v.form.customMin"